README.md

```diff
@@ -70,10 +70,14 @@ To build Clara Genomics Analysis -
 ```bash
 mkdir build
 cd build
-cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=install
+cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=install -Dcga_cuda_gen_all_arch=OFF
 make -j install
 ```
 
+NOTE : The `cga_cuda_gen_all_arch=OFF` option pre-generates optimized code only for the GPU(s) on your system.
```

Comment:
Just confirming... Our minimal supported CMake is 3.10.2 and it already had support for Volta, so this should be fine: https://gitlab.kitware.com/yrHeTaTeJlb/cmake/-/blob/v3.10.2/Modules/FindCUDA/select_compute_arch.cmake
I'm not sure if explicitly specifying 7.5 (Turing) brings performance improvements

Reply:
I ran it on my local machine which is running cmake 3.10.2, and it detected 7.5 for my RTX 2080Ti as well

Reply:
Ok, that's interesting. In code I only see 7.0, but perhaps there is some magic happening in the background...

Reply:
Yeah, I was concerned about what the code showed as well, so I wanted to test it out. And noticed that it's working fine (at least till Turing).

Reply:
Can you run `cuobjdump cudamapper/cudamapper -lelf` and see what `xy` dou you get in `*.sm_xy.cubin`?

Reply:
ELF file    1: main.sm_70.cubin
ELF file    2: main.sm_75.cubin
ELF file    3: matcher.sm_70.cubin
ELF file    4: matcher.sm_75.cubin
ELF file    5: index.sm_70.cubin
ELF file    6: index.sm_75.cubin
ELF file    7: index_gpu.sm_70.cubin
ELF file    8: index_gpu.sm_75.cubin
ELF file    9: host_cache.sm_70.cubin
ELF file   10: host_cache.sm_75.cubin
ELF file   11: minimizer.sm_70.cubin
ELF file   12: minimizer.sm_75.cubin
ELF file   13: matcher_gpu.sm_70.cubin
ELF file   14: matcher_gpu.sm_75.cubin
ELF file   15: overlapper_triggered.sm_70.cubin
ELF file   16: overlapper_triggered.sm_75.cubin
ELF file   17: hirschberg_myers_gpu.sm_70.cubin
ELF file   18: hirschberg_myers_gpu.sm_75.cubin

Reply:
Awesome!